### PR TITLE
Support custom bots

### PR DESCRIPTION
**Pull request by acheong08**
Supports fetching and using user-created bots from https://poe.com/explore_bots

Example:
```python
import poe
from os import getenv

client = poe.Client(getenv("POE_TOKEN"))

bots = client.explore_bots()

bots = client.explore_bots(end_cursor=bots["end_cursor"])

bot = bots["bots"][-1]
bot_name = bot.get("displayName")
bot_id = bot.get("botId")

print(f"Bot name: {bot_name}")

message = "Summarize the GNU GPL v3"
for chunk in client.send_message(
    bots["bots"][-1].get("displayName"), message, with_chat_break=True
):
    print(chunk["text_new"], end="", flush=True)
```

### CODE REVIEW

**Comment by ading2210** [comment]
Gonna test this myself, and if it works then I'll merge.

File: poe-api/src/poe.py
Comment (ading2210) [inline poe-api/src/poe.py:1]: It looks like you've converted everything to use 4 spaces? Please don't do that.

**Review by ading2210 — CHANGES_REQUESTED** [review]

File: poe-api/src/poe.py
Comment (acheong08) [reply poe-api/src/poe.py:1]: Oops. Automatic formatting was turned on. How to revert?

**Comment by acheong08** [comment]
I'll reset and apply the changes again

File: poe-api/src/poe.py
Comment (ading2210) [reply poe-api/src/poe.py:1]: I belive the [reindent](https://pypi.org/project/reindent/) package can do that automatically.

**Pull request closed without merging**